Description: also disable ICU C++ header-only APIs, introduced in ICU 76
 They require C++17 which we do not have, but we do not need them anyway.
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: https://github.com/qtwebkit/qtwebkit/pull/1107
Last-Update: 2025-01-09

--- a/Source/WTF/wtf/Platform.h
+++ b/Source/WTF/wtf/Platform.h
@@ -1140,6 +1140,7 @@
 /* FIXME: This does not belong in Platform.h and should instead be included in another mechanism (compiler option, prefix header, config.h, etc) */
 /* ICU configuration. Some of these match ICU defaults on some platforms, but we would like them consistently set everywhere we build WebKit. */
 #define U_SHOW_CPLUSPLUS_API 0
+#define U_SHOW_CPLUSPLUS_HEADER_API 0
 #ifdef __cplusplus
 #define UCHAR_TYPE char16_t
 #endif
